### PR TITLE
Add io.github.mlm_games.asteroids_revenge (change of app-id)

### PR DESCRIPTION
**Pull request by mlm-games**
### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by mlm-games** [comment]
bot, build io.github.mlm_games.asteroids_revenge.yml

File: io.github.mlm_games.asteroids_revenge.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      - type: file
+        url: https://raw.githubusercontent.com/mlm-games/asteroids-revenge/main/flathub_stuff/io.github.mlm_games.asteroids_revenge.desktop
+        sha256: 06d679c195fde2e8979b6924b9461e5c6c49c261bd0948973e0ec66368d4d4cf
+        dest-filename: io.github.mlm_games.asteroids_revenge.desktop
+      - type: file
+        url: https://raw.githubusercontent.com/mlm-games/asteroids-revenge/main/flathub_stuff/io.github.mlm_games.asteroids_revenge.metainfo.xml
+        sha256: 41c73b2c857d474a69ce7d096c413f45a2a282bf76a7617e84b62e6d2df20805
+        dest-filename: io.github.mlm_games.asteroids_revenge.metainfo.xml
+      - type: file
+        url: https://raw.githubusercontent.com/mlm-games/asteroids-revenge/main/flathub_stuff/icon.png
```
Comment (hfiguiere) [inline io.github.mlm_games.asteroids_revenge.yml:42]: this should NEVER be referencing a branch

File: io.github.mlm_games.asteroids_revenge.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        sha256: b8a481a2cd21bb45d7225836ceb564b7986ddacbbe3866063d2371b8d3c61abf
+        dest-filename: asteroids_revenge
+        only-arches:
+          - aarch64
+      - type: file
+        url: https://raw.githubusercontent.com/mlm-games/asteroids-revenge/main/flathub_stuff/io.github.mlm_games.asteroids_revenge.desktop
+        sha256: 06d679c195fde2e8979b6924b9461e5c6c49c261bd0948973e0ec66368d4d4cf
+        dest-filename: io.github.mlm_games.asteroids_revenge.desktop
+      - type: file
+        url: https://raw.githubusercontent.com/mlm-games/asteroids-revenge/main/flathub_stuff/io.github.mlm_games.asteroids_revenge.metainfo.xml
```
Comment (hfiguiere) [inline io.github.mlm_games.asteroids_revenge.yml:38]: this should NEVER be referencing a branch

File: io.github.mlm_games.asteroids_revenge.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        only-arches:
+          - x86_64
+      - type: file
+        url: https://github.com/mlm-games/asteroids-revenge/releases/download/0.10.4/asteroids-revenge-linux-aarch64.arm64
+        sha256: b8a481a2cd21bb45d7225836ceb564b7986ddacbbe3866063d2371b8d3c61abf
+        dest-filename: asteroids_revenge
+        only-arches:
+          - aarch64
+      - type: file
+        url: https://raw.githubusercontent.com/mlm-games/asteroids-revenge/main/flathub_stuff/io.github.mlm_games.asteroids_revenge.desktop
```
Comment (hfiguiere) [inline io.github.mlm_games.asteroids_revenge.yml:34]: this should NEVER be referencing a branch

**Comment by hfiguiere** [comment]
bot, build io.github.mlm_games.asteroids_revenge

**Comment by mlm-games** [comment]
bot, build io.github.mlm_games.asteroids_revenge

**Comment by mlm-games** [comment]
Changed the refernece to a commit and it works fine

**Pull request closed without merging**